com.blunix.git.currency_converter.yml

```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.blunix.git.currency_converter
```

Comment:
Do you own blunix.com or is it just hosting the gitlab instance like gitlab.com?

Reply:
The app was developed by blunix company that owns blunix.com

Reply:
then it should be `com.blunix` without the `git. But you say **was**. So it's in the past, who control the domain?

Reply:
The app was developed by blunix company that owns blunix.com

Reply:
Blunix does currently control the domain. I decided to add git because the homepage of the app is hosted on git.blunix.com.  
If it aligns better with the flatpak conventions then I will change it to com.blunix. Would you recommend to proceed with the change?

Reply:
yes it should be changed.

Reply:
I changed the name. Is it alright or should I create a new pull request from a different branch?

Reply:
no

the branch name is irrelevant.